src/main/java/com/katyat/classesandobjects/autoservicetask/Tool.java

```diff
@@ -0,0 +1,18 @@
+package com.katyat.classesandobjects.autoservicetask;
+
+public class Tool {
+
+    boolean needsElectricity;
+    boolean isElevator;
+    boolean isUsed;
+    long uniqueCode;
+
+    Tool(){
+        System.out.println("A fost creat un instrument despre care nu se stie nimic");
+    }
+
+    Tool(long uniqueCodeParam){
+        uniqueCode = uniqueCodeParam;
```

Comment:
missing "this." for "uniqueCode" property

Reply:
Andrei deleted "this." for "uniqueCode" and left only (uniqueCode = uniqueCodeParam;)